### DISCUSSION TITLE
Multiple workspaces 💼

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#3646] [Ideas]
Design hasn't been done yet.
Not a top priority.

**Comment by FelixMalfait** [comment]
Related discussion: https://discord.com/channels/1130383047699738754/1146466959676936192/1199728041501405267

<img width="895" alt="Screenshot 2024-01-25 at 15 52 23" src="https://github.com/twentyhq/twenty/assets/6399865/e20ec249-1325-4a15-b828-06b5de3fed2d">

**Comment by Bonapara** [comment]
Old title: After login, I can select workspace if I'm logged into more than one workspace

**Comment by martmull** [comment]
Design -> https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=1406-44830&mode=design&t=4pZxqigV9aaKzc5U-0


https://github.com/twentyhq/twenty/assets/29927851/8e5a539b-2abb-4e65-af77-1dad8209bb1b

**Comment by FelixMalfait** [comment]
This was completed
